gobblin-cluster/src/main/java/org/apache/gobblin/cluster/FsJobConfigurationManager.java

```diff
@@ -88,6 +88,12 @@ public void run() {
     }, 0, this.refreshIntervalInSeconds, TimeUnit.SECONDS);
   }
 
+  @Override
+  protected void shutDown() throws Exception {
```

Comment:
Interesting. What happens in the absence of shutdown() method?  It was a little confusing from the PR description - if AM is stopped and the JobConfigManager continues to check for job specs, does it mean that the AM is prevented from exiting? Is there any impact on the Gobblin-on-Yarn application because of this?

Reply:
No, it does not affect the exiting. It will stop after the whole container is existed. Just in another issue where AM stuck after existing, I notice this thread also not been exited, so make this change.